### PR TITLE
Use C++14 to allow building for Electron 11 on MacOS

### PR DESCRIPTION
**Pull request by sergei-dyshel**
Fixes https://github.com/tree-sitter/node-tree-sitter/issues/82.

### CODE REVIEW

**Comment by Symbolk** [comment]
@maxbrunsfeld Could this be merged and released? A timely change for me.

**Comment by jinu-jang** [comment]
This is just necessary for recent electron versions (v11, v12)
@sergei-dyshel thanks for the fix. This at least got me unblocked for now

**Comment by Symbolk** [comment]
Right, for those who use electron v11+, here is the manual instruction to rebuild this module:

https://github.com/Symbolk/somanyconflicts#known-issues

**Comment by DeeDeeG** [comment]
:+1:, and I think not merging this might slow progress on using Electron 11 in the Atom text editor. Might have to fork this package for Atom just to get it building in Electron 11.

**Comment by sadick254** [comment]
@maxbrunsfeld What could be holding back this PR from getting merged? As @DeeDeeG has stated this currently blocks Atom from upgrading to electron version 11.4.7.

File: binding.gyp [hunk 1]
```diff
@@ -31,7 +31,7 @@
         "-std=c++0x",
       ],
       'xcode_settings': {
-        'CLANG_CXX_LANGUAGE_STANDARD': 'c++11',
+        'CLANG_CXX_LANGUAGE_STANDARD': 'c++14',
```
Comment (aminya) [inline binding.gyp:34]:
```suggestion
        'CLANG_CXX_LANGUAGE_STANDARD': 'c++17',
```

**Review by aminya — CHANGES_REQUESTED** [review]
Why not c++17?

**Comment by Symbolk** [comment]
After being torture by native module (node-gyp and electron-rebuild) for months, I finally switched to wasm of treesitter, and it seems working fine: https://github.com/Symbolk/somanyconflicts

Maybe this solution helps some of you.

**Comment by verhovsky** [comment]
This PR can be closed because #92 (merged) did the same thing.

**Pull request closed without merging**